haiku-apps/bemobile/bemobile-0.9.recipe

```diff
@@ -40,6 +42,6 @@ BUILD()
 INSTALL()
 {
 	mkdir -p $appsDir
-	cp objects/BeMobile $appsDir
-	addAppDeskbarSymlink $appsDir/BeMobile
+	cp objects/BeMobile $appsDir/BeMobile/BeMobile
+	addAppDeskbarSymlink $appsDir/BeMobile/BeMobile
```

Comment:
I would revert these 2 lines, no need to have BeMobile in $appDir/BeMobile, having it in $appsDir should be enough